### ISSUE TITLE
Reduce size of the 2.2uH inductor pad

### ISSUE THREAD

**Issue by brandoaire** [spark/electron#26]
![screenshot 2015-08-28 14 13 35](https://cloud.githubusercontent.com/assets/4020511/9557203/02545234-4d8f-11e5-8d51-f856854f1971.png)

Feedback from beta unit run, line 2

**Comment by brandoaire** [comment]
From @ericyuanyong 
According to the DFM report from ABO. We need to replace the footprint of Diode(U$3) to avoid the misalign issue for the diode when go through re-flow oven. But In that this footprint was also used on Core and Photon, so should we change it this time.

As the inductor's issue, I remember the reason we changed the footprint of this inductor(U$4) was because we want to have another choice for inductor. So we had decided only use part number for this inductor?
